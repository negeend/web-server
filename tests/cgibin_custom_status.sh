cd ..
python3 webserv.py config.cfg &
PID=$!
cd tests
curl -I localhost:8070/cgibin/custom_status.py 2> /dev/null | grep 'HTTP/1.1' | diff - cgibin_custom_status.out || echo "Failed CGI custom status test"
kill $PID
