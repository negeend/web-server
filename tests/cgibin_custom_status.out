HTTP/1.1 123 Random
